src/components/boards/Boards.jsx

```diff
@@ -0,0 +1,114 @@
+import React, {useState, useEffect} from "react"
+import { items } from "../AllItems/AllItems"
+import "./Board.css";
+function Boards ({setSelectedItem, setSelectedCategory}){
+
+    const [selecetBoerd, setSelecetBoerd] = useState("")
+
+    const allIems = items.map(item=> { return (item) })
+    // const catagories = items.map(item=> { return (item.type) })
```

Comment:
typo in categories, also if we're not going to use this line of code, it should be deleted instead of commented out